kobo/apps/help/forms.py

```diff
@@ -1,10 +1,22 @@
 # coding: utf-8
 import os
 
+from django.forms import ModelForm, ValidationError
 from django.urls import reverse
 from markdownx.forms import ImageForm
 
-from .models import InAppMessageFile
+from .models import InAppMessage, InAppMessageFile
+
+
+class InAppMessageForm(ModelForm):
+
+    class Meta:
+        model = InAppMessage
+        exclude = ['generic_related_objects']
+
+    def clean(self):
+        cleaned_data = super().clean()
+        return cleaned_data
```

Comment:
Is this needed? It looks like it's just invoking `ModelForm.clean()`.

Reply:
Well. Nothing to say for my own defense 😛